packages/neos-ui-editors/src/Editors/Reference/style.css

```diff
@@ -0,0 +1,12 @@
+.referenceOption {
+    padding: var(--halfSpacing);
+    box-sizing: content-box;
+    height: var(--goldenUnit);
+    line-height: 20px;
+}
+
+.referenceOption__link {
+    color: var(--brandColorsContrastBright);
+    display: block;
+    font-size: 12px;
```

Comment:
I'm a bit concerned how we use font-sizes around the codebase.
I will create an issue to gather all of these special font usages into a separate react-ui-components component, and use them to share styles. Otherwise our styles will quickly turn into a mess.

Reply:
:+1: